python/py-click-didyoumean/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-click-didyoumean
+version             0.0.3
+revision            0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.versions     38 39
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         Enable git-like did-you-mean feature in click
+long_description    ${description}
+
+homepage            https://github.com/click-contrib/click-didyoumean
+
+checksums           rmd160  217545f3e2d4d724d721b18b3db24c189f195fa2 \
+                    sha256  112229485c9704ff51362fe34b2d4f0b12fc71cc20f6d2b3afabed4b8bfa6aeb \
+                    size    2562
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+
+    depends_run-append      port:py${python.version}-click
```

Comment:
typically within the subports we set `livecheck.type none` so that only for the `py-<portname>` the livecheck is active

Reply:
OK, understood; added.